rasa/shared/nlu/training_data/training_data.py

```diff
@@ -181,9 +196,14 @@ def intents(self) -> Set[Text]:
         """Returns the set of intents in the training data."""
         return {ex.get(INTENT) for ex in self.training_examples} - {None}
 
+    @lazy_property
+    def actions(self) -> Set[Text]:
```

Comment:
Can we rename it to `action_names`? Primarily because end-to-end is going to get it `action_texts` as well and then `actions` as a property will be ambiguous.